virtcontainers/qemu.go

```diff
@@ -598,10 +598,14 @@ func (q *qemu) startSandbox(timeout int) error {
 		// connection with QEMU closes.  Therefore we do not keep track
 		// of this child process after returning from this function.
 		sourcePath := filepath.Join(kataHostSharedDir, q.id)
-		cmd := exec.Command(q.config.VirtioFSDaemon,
-			"-o", "vhost_user_socket="+sockPath,
-			"-o", "source="+sourcePath,
-			"-o", "cache="+q.config.VirtioFSCache)
+		args := []string{
+			"-o", "vhost_user_socket=" + sockPath,
+			"-o", "source=" + sourcePath,
+			"-o", "cache=" + q.config.VirtioFSCache}
+		if q.config.Debug {
+			args = append(args, "-d")
```

Comment:
I'm not sure I understand this fully - don't we want to add this `-d` option always to detect the scenario you are describing?

Reply:
Good point! We can use `-f` flag to avoid getting too many debug outputs.

Reply:
updated. ptal.